samza-core/src/test/java/org/apache/samza/context/TestContextImpl.java

```diff
@@ -63,11 +67,28 @@ public void testGetMissingApplicationTaskContext() {
   }
 
   private static Context buildWithApplicationContainerContext(ApplicationContainerContext applicationContainerContext) {
+<<<<<<< HEAD
+    return buildWithApplicationContext(applicationContainerContext, mock(ApplicationTaskContext.class));
+  }
+
+  private static Context buildWithApplicationTaskContext(ApplicationTaskContext applicationTaskContext) {
+    return buildWithApplicationContext(mock(ApplicationContainerContext.class), applicationTaskContext);
+  }
+
+  private static Context buildWithApplicationContext(ApplicationContainerContext applicationContainerContext,
+      ApplicationTaskContext applicationTaskContext) {
+    return new ContextImpl(mock(JobContext.class),
+        mock(ContainerContext.class),
+        mock(TaskContext.class),
+        applicationContainerContext,
+        applicationTaskContext);
+=======
```

Comment:
Let's resolve this merge conflict.

Reply:
Oops, missed that. Thanks for pointing it out.